pg-extend/src/lib.rs

```diff
@@ -273,13 +273,20 @@ macro_rules! pg_create_stmt_bin {
         #[cfg(target_os = "macos")]
         const DYLIB_EXT: &str = "dylib";
 
-        #[cfg(target_os = "windows")]
-        const DYLIB_EXT: &str = "dll";
+        #[cfg(unix)]
+        fn main() {
+            let lib_name = env!("CARGO_PKG_NAME").replace("-", "_");
+
+            let lib_path = env::args().nth(1).unwrap_or(format!("target/release/lib{}.{}", lib_name, DYLIB_EXT));
 
+            $( println!("{}", lib::$func(&lib_path)); )*
+        }
+        
+        #[cfg(windows)]
         fn main() {
-            const LIB_NAME: &str = env!("CARGO_PKG_NAME");
+            let lib_name = env!("CARGO_PKG_NAME").replace("-", "_");
```

Comment:
When building, the resulting files will always use underscores, so we need to replace dashes with underscores, otherwise there's a mismatch